src/app/pages/datasets/components/dataset-management/dataset-management.component.ts

```diff
@@ -125,16 +125,7 @@ export class DatasetsManagementComponent implements OnInit, AfterViewInit, OnDes
     private slideIn: IxSlideInService,
     private store$: Store<AppState>,
     @Inject(WINDOW) private window: Window,
-  ) {
-    this.router.events
-      .pipe(filter((event) => event instanceof NavigationStart), untilDestroyed(this))
-      .subscribe(() => {
-        this.layoutService.pageHeaderUpdater$.next(this.pageHeader);
-        if (this.router.getCurrentNavigation().extras.state?.hideMobileDetails) {
-          this.closeMobileDetails();
```

Comment:
Removing this breaks the layout on mobile. On mobile the details card visibility is toggled. With this code gone, users cannot get out of the details view and back to the table


In the screenshot the back arrow is no longer working

<img width="393" alt="image" src="https://user-images.githubusercontent.com/29865743/234293997-3fb0a279-7a00-4fa3-b69a-555736ec0ec0.png">